h/migrations/versions/f052da9df33b_make_group_organization_id_not_nullable.py

```diff
@@ -0,0 +1,23 @@
+"""
+Make group.organization_id not nullable.
+
+Revision ID: f052da9df33b
+Revises: 8a7f31c4525d
```

Comment:
This comment needs updating to match the `down_revision` below.

Reply:
Fixed